### PR TITLE
Update elements reference - Closes #552

### LINKED ISSUE #552: Update Elements reference
## Which content is outdated?
- [x] Changed commands of existing packages
- [x] New packages
- [x] Index page

## What should be the new content?
A brief reference of Lisk Elements' packages.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #552

### CODE REVIEW

File: modules/ROOT/pages/reference/lisk-elements/transactions.adoc [hunk 1]
```diff
@@ -338,97 +324,202 @@ If not provided at the creation, the transaction can be signed with the second p
...
+  confirmations: undefined,
+  blockId: undefined,
+  height: undefined,
+  receivedAt: undefined,
+  asset: { data: 'Hello Lisk!', recipientId: '123L', amount: 15000000n }
+}
+ */
+----
+
+<1> The network identifier of the network, you want to post the transaction to.
```
Comment (chris529) [inline modules/ROOT/pages/reference/lisk-elements/transactions.adoc]:
```suggestion
<1> The network identifier of the network, where you wish to post the transaction.
```

File: modules/ROOT/pages/reference/lisk-elements/transactions.adoc [hunk 2]
```diff
@@ -338,97 +324,202 @@ If not provided at the creation, the transaction can be signed with the second p
...
+<2> Enter here the 12 word mnemonic passphrase for an account
+
+After the transaction is signed, the `signatures`, `senderPublicKey` and `networkIdentifier` are added to the transaction object.
+
+It is also possible to get the minimum fee for this particular trasnaction, to be accepted by the network:
```
Comment (chris529) [inline modules/ROOT/pages/reference/lisk-elements/transactions.adoc]:
```suggestion
It is also possible to get the minimum fee for this particular transaction, to be accepted by the network:
```

File: modules/ROOT/pages/reference/lisk-elements/transactions.adoc [hunk 3]
```diff
@@ -649,51 +740,28 @@ verifySenderPublicKey(id, sender, publicKey)
 `undefined`: if the public key is verified.
  A `TransactionError` is thrown, if the verification fails.
 
-=== verifyAmountBalance
+=== verifyMinRemainingBalance
 
-Verifies if an account has enough balance to pay the amount and the fee of a transaction.
+Verifies if an account has enough remaining balance in the account.
```
Comment (chris529) [inline modules/ROOT/pages/reference/lisk-elements/transactions.adoc]:
```suggestion
Verifies if the remaining balance in the account is sufficient.
```

File: modules/ROOT/pages/reference/lisk-elements/p2p.adoc [hunk 4]
```diff
@@ -70,7 +70,7 @@ It provides a simple interface to send, request, broadcast information and many
 - `p2p.getConnectedPeers()`: get all the connected peers that are connected to your node in the network.
 - `p2p.getDisconnectedPeers()`: get all the disconnected peers that are part of the network but not connected to you.
 - `p2p.request(packet: P2PRequestPacket)`: request information from the network that will run the peer selection and finds an appropriate peer for you to request information.
-- `p2p.send(message: P2PMessagePacket)`: sends information to 16 connected peers choosen by peer selection for send.
+- `p2p.send(message: P2PMessagePacket)`: sends information to 16 connected peers chosen by peer selection for send.
```
Comment (chris529) [inline modules/ROOT/pages/reference/lisk-elements/p2p.adoc]: "sends information to 16 connected peers chosen by peer selection for send".

This sentence  doesn't seem to be clear to me..?  Perhaps we could say " sends information to 16 connected peers chosen by  the peer selection process, which is required for the sending procedure". ?

**Review by chris529 — CHANGES_REQUESTED** [review]
Looks good, minor points, and 1 sentence that didn't seem to be clear to me.. :-)

File: modules/ROOT/pages/reference/lisk-elements/p2p.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/reference/lisk-elements/p2p.adoc]: Its coming from here: https://github.com/LiskHQ/lisk-sdk/tree/development/elements/lisk-p2p
@ishantiw suggested we can change it to 
```suggestion
- `p2p.send(message: P2PMessagePacket)`: sends information to 16 connected peers chosen by the peer selection function.
```

File: modules/ROOT/pages/reference/changelog.adoc [hunk 5]
```diff
@@ -1,5 +1,6 @@
 = Developer changelog
 Mona Bärenfänger <mona@lightcurve.io>
+:description: This section contains the devloper changelog information from the transaction types and updated genesis block for the devnet, to the database changes including the peers, delegates and nodes.
```
Comment (chris529) [inline modules/ROOT/pages/reference/changelog.adoc]:
```suggestion
:description: This section contains the developer changelog information from the transaction types and updated genesis block for the devnet, to the database changes including the peers, delegates and nodes.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Perfect...just one minor typo :-)

**Review by chris529 — APPROVED** [review]
Perfect.. 👍

**Pull request merged**